webapp/templates/jury/partials/submission_list.html.twig

```diff
@@ -72,11 +72,27 @@
                 {%- set link = path('jury_submission', {submitId: submission.submitid}) %}
             {%- endif %}
 
+            {% set result = 'queued' %}
+            {% if submission.submittime > submission.contest.endtime %}
+                {% if submission.judgings.first is not empty and submission.judgings.first.result is not empty %}
+                    {% set result = submission.judgings.first.result %}
+                {% endif %}
+            {% elseif submission.judgings.first is empty or submission.judgings.first.result is empty %}
+                {% if submission.judgehost %}
+                    {% set result = 'judging' %}
+                {% else %}
+                    {% set result = 'queued' %}
+                {% endif %}
+            {% else %}
+                {% set result = submission.judgings.first.result %}
+            {% endif %}
```

Comment:
this is ugly, but I don't see an obvious way to make it clearer

Reply:
Yes I had the same issue.... We could put this in a twig function and reuse it? But then too-late is hard

Reply:
Maybe we should not be doing this in twig in the first place?
Something for the controller to determine what the 'verdict' is,including juding and queued. And also add a too-late flag as well that twig can act on?

Reply:
Yes this might be better. I will try it

Reply:
Gave it a try